docs/source/updating_capabilities.md

```diff
@@ -0,0 +1,341 @@
+the version of a component (for example, going from one version of a binary to
+the next version). The term “update,” on the other hand, refers not to versions
+but to configuration changes, such as updating a channel configuration or a
+deployment script. As there is no data migration, technically speaking, in Fabric,
+we will not use the term "migration" or "migrate" here.
+
+## Overview
+
+In this tutorial, we will show the process for updating capabilities in all of the
+parts of the configuration of both the ordering system channel and any application
+channels.
+
+Whether you will need to update every part of the configuration for all
+of your channels will depend on the contents of the latest release as well as
+your own use case. For more information, check out [Upgrading to the latest version of Fabric](./upgrade_to_newest_version.html).
+Note that it may be necessary to update to the newest capability levels before
+using the features in the latest release, and it is considered a best practice
+to always be at the latest binary versions and capability levels.
+
+Because updating the capability level of a channel involves the configuration
+update transaction process, we will be relying on our [Updating a channel configuration](./config_update.html)
+topic for many of the commands.
+
+As with any channel configuration update, updating capabilities is, at a high
+level, a three step process (for each channel):
+
+1. Get the latest channel config
+2. Create a modified channel config
+3. Create a config update transaction
+
+We will enable these capabilities in the following order:
+
+1. [Orderer system channel](#orderer-system-channel-capabilities)
+
+  a. Orderer group
+  b. Channel group
+
+2. [Application channels](#enable-capabilities-on-existing-channels)
+
+  a. Orderer group
+  b. Channel group
+  c. Application group
+
+While it is possible to edit multiple parts of the configuration of a channel
+at the same time, in this tutorial we will show how this process is done incrementally.
+In other words, we will not bundle a change to the `Orderer` group and the `Channel`
+group of the system channel into one configuration change. This is because not
+every release will have both a new `Orderer` group capability and a `Channel`
+group capability.
+
+Note that in production networks, it will not be possible or desirable for one
+user to be able to update all of these channels (and parts of configurations)
+unilaterally. The orderer system channel, for example, is administered exclusively
+by ordering organization admins (though it is possible to add peer organizations
+as ordering service organizations). Similarly, updating either the `Orderer` or
+`Channel` groups of a channel configuration requires the signature of an ordering
+service organization in addition to peer organizations. Distributed systems
+require collaborative management.
+
+### Prerequisites and considerations
```

Comment:
I would put this section above the overview so that the capabilities overview info and the json file are next to each other.

Reply:
I like this suggestion. It's a bit galling to have something before the overview but I think it works for this doc.